fe/fe-core/src/main/java/org/apache/doris/catalog/ColocateTableIndex.java

```diff
@@ -206,13 +207,14 @@ public void addBackendsPerBucketSeqByTag(GroupId groupId, Tag tag, List<List<Lon
         }
     }
 
-    public void markGroupUnstable(GroupId groupId, boolean needEditLog) {
+    public void markGroupUnstable(GroupId groupId, String reason, boolean needEditLog) {
```

Comment:
Using error codes or string constants may be better than literal values

Reply:
Error code is not necessary. Because currently this reason is only for human reading.
I tried to use string constants. But reason like:
`get unhealthy tablet %d in colocate table. status: %s`
is dynamic changed.
So not all reasons can be constants